pkgs/development/python-modules/pep8-naming/default.nix

```diff
@@ -12,10 +16,27 @@ buildPythonPackage rec {
     sha256 = "0937rnk3c2z1jkdmbw9hfm80p5k467q7rqhn6slfiprs4kflgpd1";
   };
 
+  patches = [
```

Comment:
```
       > applying patch /nix/store/gc9hjsi1ibrb0qgikwpa2bp670bfrnq6-6d62db81d7967e123e29673a4796fefec6f06d26.patch
       > patching file run_tests.py
       > Hunk #1 FAILED at 89.
       > 1 out of 1 hunk FAILED -- saving rejects to file run_tests.py.rej
       For full logs, run 'nix log /nix/store/spb52ia2a7spra1fvddz8wdy483ydlgy-python3.7-pep8-naming-0.4.1.drv'
```

Reply:
Please read your build logs more carefully. The prospector package is in a broken state and overwrites pep8-naming. Even after fixing that overwrite it is still broken, because it requires more pinning and is built on top of python2.

```
ERROR: Could not find a version that satisfies the requirement pylint-django==2.0.12 (from prospector)
ERROR: No matching distribution found for pylint-django==2.0.12
```